README.md

```diff
@@ -55,11 +55,13 @@ Spaceport provides several end-user applications.
 ## Scenarios
 
 ### Send test events
-For sending test events use `sender.jar`.
+For sending test events use next commands:
 
-Example of properties file: [application.properties](etc/properties/sender/application.properties).
+ 1. docker pull vstk/hercules-gateway-client:0.38.0-SNAPSHOT
```

Comment:
IIRC, you can run `docker run` directly, without pulling image manually. Try it.

Reply:
It's true, checked.